webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/annotation/BratAnnotationEditor.java

```diff
@@ -163,11 +169,11 @@ public void onEndRequest(RequestCycle aCycle)
                         BratAnnotationEditor.this.getModelObject().getAction().clearUserAction();
                     }
                 });
-
+                
                 // Load the CAS if necessary
                 // Make sure we load the CAS only once here in case of an annotation action.
                 boolean requiresCasLoading = SpanAnnotationResponse.is(action)
-                        || ArcAnnotationResponse.is(action) || GetDocumentResponse.is(action);
+                        || ArcAnnotationResponse.is(action) || GetDocumentResponse.is(action) || "doAction".equals(action);
```

Comment:
I think a response class should be implemented it's `is(...)` method used here, cf. GetDocumentResponse.